support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -622,7 +624,7 @@ export function CheckoutComponent({
 				<Box
 					cssOverrides={[
 						shorterBoxMargin,
-						isAdLite ? lengthenBoxMargin : css``,
+						isAdLite || !isRecurringContribution ? lengthenBoxMargin : css``,
```

Comment:
Could this be simplified?

```suggestion
						!isRecurringContribution ? lengthenBoxMargin : css``,
```

I think `!isRecurringContribution` will be true for Ad-Lite so probably no need to cover that case separately. And if the `||` is removed the ternary can be un-negated/swapped to:

```
    isRecurringContribution ? css`` : lengthenBoxMargin,
```

Reply:
Good spot, the suggestion is neater so I'll commit it 👍